cluster-up/cluster/kind/manifests/kind.yaml

```diff
@@ -4,5 +4,9 @@ containerdConfigPatches:
 - |-
   [plugins."io.containerd.grpc.v1.cri".registry.mirrors."registry:5000"]
     endpoint = ["http://registry:5000"]
+  [plugins."io.containerd.grpc.v1.cri".registry.mirrors."quay.io"]
+    endpoint = ["http://docker-mirror-proxy.kubevirt-prow.svc:3128"]
+  [plugins."io.containerd.grpc.v1.cri".registry.mirrors."docker.io"]
+    endpoint = ["http://docker-mirror-proxy.kubevirt-prow.svc:3128"]
 nodes:
 - role: control-plane
```

Comment:
Is this configuration just applied to the control-plane nodes? is separate proxy configuration required for workers?

Reply:
It will apply on each cluster node, I verified by looking at each node`/etc/containerd/config.toml` file
and saw changes were reflected.

Reply:
Ok thats great.